clients/portal/patientConfig.go

```diff
@@ -0,0 +1,90 @@
+// Package portal is a client module to support server-side use of the Diabeloop
+// service called user-api.
+//
+// This file contains the definitions for the patient config
+package portal
```

Comment:
should it be package "clients"? We seem to have some discrepancies already. For example, gatekeeper is in package "clients" where shoreline is in package "shoreline". Not sure why and what we want to achieve here, just asking.

Reply:
> 
> 
> should it be package "clients"? We seem to have some discrepancies already. For example, gatekeeper is in package "clients" where shoreline is in package "shoreline". Not sure why and what we want to achieve here, just asking.

Not 100% sure, but normally I just followed the golang rules here.
golint seems happy with that.

Reply:
https://golang.org/doc/effective_go.html#package-names

Reply:
my remark was about the organisation of the different clients into namespaces. Some are in namespace "clients", some others have their own namespace. This can be confusing when using the library

Reply:
I am OK validating the PR like this but this might be something we can look at later